public/app/features/explore/TraceView/components/TracePageHeader/TracePageHeader.tsx

```diff
@@ -234,11 +254,52 @@ export default function TracePageHeader(props: TracePageHeaderEmbedProps) {
     </h1>
   );
 
+  const newTitle = (
+    <h1 className={cx(styles.TracePageHeaderTitle)}>
+      <TraceName traceName={getTraceName(trace.spans)} />
+      <small>
+        <span className={styles.divider}>|</span>
+        {formatDuration(trace.duration)}
+      </small>
+    </h1>
+  );
+
+  const method =
+    trace.spans[1] &&
+    trace.spans[1].tags.filter((tag) => {
+      return tag.key === 'http.method';
+    });
```

Comment:
Just updating the way we choose a span from which to get the method, status and url.. will update PR :)